presto-geospatial/src/main/java/com/facebook/presto/geospatial/BingTileFunctions.java

```diff
@@ -206,8 +206,8 @@ private static BingTile latitudeLongitudeToTile(double latitude, double longitud
         double y = 0.5 - Math.log((1 + sinLatitude) / (1 - sinLatitude)) / (4 * Math.PI);
 
         int mapSize = mapSize(zoomLevel);
-        int tileX = (int) clip(x * mapSize + 0.5, 0, mapSize - 1);
```

Comment:
My understanding is when `x*mapSize` is around `n.5`, `x * mapSize + 0.5` will be around `n+1` which maybe round to `n` or `n+1` because of floating number precision? 

But if we remove `+0.5`, will the similar situation happens when `x*mapSize` is around an integer? :)